catalog/urls.py

```diff
@@ -18,12 +18,6 @@
 ]
 
 
-urlpatterns += [   
```

Comment:
This code is duplicated so removing this part will leave one more block about renew a book, you can verify it here https://github.com/mdn/django-locallibrary-tutorial/blob/150da77b13d24b0ef5c5ebfc7741830700c41c4c/catalog/urls.py looking at full file. Don't know why but this part was written twice in this file.